### ISSUE TITLE
/opt/bin/xochitl script issues

### ISSUE THREAD

**Issue by Havner** [matteodelabre/toltec#908]
**Describe the issue**
2 small issues in the script:

1. The `is-active` function doesn't exist here
https://github.com/toltec-dev/toltec/blob/stable/package/xochitl/xochitl#L24
2. The script masks non existing service (e.g. rm-sync):
https://github.com/toltec-dev/toltec/blob/stable/package/xochitl/xochitl#L20C1-L22C3
And nothing unmasks it when uninstalling as the unmask option have different conditions in the postremove:
https://github.com/toltec-dev/toltec/blob/stable/package/xochitl/package#L110C1-L111C41

Not putting more info as this has been discussed in the discord and I think the code is clear enough to see the issues.

**Comment by palfrey** [comment]
* `is-active` comes from https://github.com/toltec-dev/build/blob/main/toltec/hooks/install_lib.py#L71
* https://developer.remarkable.com/documentation/xochitl refers to `rm-sync` so I think it probably exists on some versions (not my rm1, but maybe rm2's?)

**Comment by Eeems** [comment]
> * `is-active` comes from https://github.com/toltec-dev/build/blob/main/toltec/hooks/install_lib.py#L71
> 
> * https://developer.remarkable.com/documentation/xochitl refers to `rm-sync` so I think it probably exists on some versions (not my rm1, but maybe rm2's?)

Correct, is-active needs to be inlined, which I've had done several times in PRs that have not been merged. rm-sync is on later OS versions and replaced sync. I believe #955 will address this.
